frontend/src/services/Platform.ts

```diff
@@ -10,3 +10,8 @@ export function isElectron() {
   const userAgent = navigator.userAgent.toLowerCase()
   return userAgent.includes(' electron/')
 }
+
+export function isMac() {
```

Comment:
Don't you mean to just use `Environment` class like above?

Reply:
The environment is different... (electron vs browser) this could be considered the platform (mac, windows, linux) maybe. I don't want to try and detect all those right now since we don't need it yet. If we do we can update.